packaging/prerelease/s3_index.sh

```diff
@@ -44,6 +44,3 @@ echo "Checking if we need to promote a release for testing ($platform)"
 
 echo "Checking if we need to promote a release ($platform)"
 "$release_bin" promote-releases --bucket-name="$bucket_name" --platform="$platform"
-
-echo "Linking latest ($platform)"
-"$release_bin" latest --bucket-name="$bucket_name" --platform="$platform"
```

Comment:
Oops, I was so wrong. The Linux build [calls into this script](https://github.com/keybase/client/blob/master/packaging/linux/build_and_push_packages.sh#L123), and depended on this line. Deleting it resulted in https://keybase.atlassian.net/browse/CORE-3077. Totally my fault.